src/main/java/algorithmRunner/AlgorithmRunner.java

```diff
@@ -7,9 +7,29 @@
 
 import java.util.List;
 
+import graph.VertexLists;
+
 public class AlgorithmRunner {
 
-    public static void runAlgorithm(RunResult result, Graph hostGraph, Graph patternGraph) {
+    public static void runAlgorithm(RunResult result, Graph hostGraph, Graph patternGraph,boolean labelled) {
+if (labelled) {
+    // If labelled, run the labelled subgraph counting algorithm
+    LabelledSubgraphCountingAlgorithm lSa = new LabelledSubgraphCountingAlgorithm(hostGraph, patternGraph, hostGraph.getHighestDegVertices(0));
```

Comment:
we still want to use the ParameterValueOptimiser to work out which vertices should be the high-degree vertices (see lines 44-49 below)

Reply:
It would also be useful to update RunResult so that we can say whether we counted labelled or unlabelled copies (could actually include both, since when we've counted unlabelled copies, we also know how many labelled copies there are)